### PR TITLE
[WIP] Added Ant Missions

### PR DESCRIPTION
**Pull request by johnweven**
I've added the four Ant Missions, but as of right now, there are no Lua scripts for them. I'm posting this PR so that my work can be visible and so others can help me out with the Lua scripts more easily.

Part of solving https://github.com/OpenRA/OpenRA/issues/4989

**Work-in-progress, do not merge!**

### CODE REVIEW

File: mods/ra/maps/ant-1/ant-1.lua [hunk 1]
```diff
@@ -0,0 +1,43 @@
+--[[
+   Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
```
Comment (reaperrr) [inline mods/ra/maps/ant-1/ant-1.lua]: Minor nit: 2018.

**Comment by johnweven** [comment]
Good news! I imported the official ant maps from Red Alert into OpenRA with the Utility tool, so all the maps are complete! Hopefully nobody has done any work on here because I'm going to reorganize my maps and swap them out with the imported ones.

**Comment by Mailaender** [comment]
What you should do is get the first ant mission going. It should be easy to script. Have a look at [survival01.lua](https://github.com/OpenRA/OpenRA/blob/bleed/mods/ra/maps/survival01/survival01.lua) which has a mission timer and enemy reinforcements.

Also try to avoid junk in the commit messages.

File: mods/ra/maps/ant-01/ant-01.lua [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+InitPlayers = function()
+	player = Player.GetPlayer("Spain")
+	ussr = Player.GetPlayer("USSR")
+	end)
```
Comment (Mailaender) [inline mods/ra/maps/ant-01/ant-01.lua]: >Checking for Lua syntax errors...
>luac: mods/ra/maps/ant-01/ant-01.lua:13: unexpected symbol near ')'

**Comment by pchote** [comment]
This has been in the PR queue for the better part of a year with no progress, so closing as part of a clean up of stale PRs.

**Pull request closed without merging**